src/sensor_fusion.c

```diff
@@ -230,7 +199,7 @@ double **compute_principal_component(struct support_degree_matrix *spd,
         printf("ERROR: Failed to allocate memory at %s\n", __func__);
         return NULL;
     }
-    printf("INFO: Compute support degree matrix\n");
+    printf("INFO : Compute support degree matrix\n");
```

Comment:
It would be clear if the value of arrptr[I][j] is printed on the same line here instead of a new line.

Reply:
removed next line character o print support degree matrix on same line in the latest fix